R/C-flux/cleaning.R

```diff
@@ -1,14 +1,34 @@
-# library(plyr) #why do I need it? maybe I don't
+# This script is to clean raw data from various loggers into one datafile with calculated fluxes
 library(tidyverse)
 source("https://raw.githubusercontent.com/jogaudard/common/master/fun-fluxes.R")
```

Comment:
This should also be a function inside ThreeD otherwise it is not reproducible. Either copy function inside ThreeD repo or make a package which can be downloaded.

Reply:
I think it is reproducible (but not elegant I agree). The repo is public and source() is fetching the script from anywhere. But at some point I'll need to write a package, just no time now.